pwiz_tools/Shared/Common/Chemistry/MzTolerance.cs

```diff
@@ -60,16 +63,32 @@ public MzTolerance(double value = 0, Units units = Units.mz)
             return 0;
         }
 
+        public static implicit operator MzTolerance(double tolerance)
+        {
+            return new MzTolerance(tolerance);
+        }
+
         /// <summary>returns true iff a is in (b-tolerance, b+tolerance)</summary>
-        public static bool IsWithinTolerance(double a, double b, MzTolerance tolerance)
+        public bool IsWithinTolerance(double a, double b)
         {
-            return (a > b - tolerance) && (a < b + tolerance);
+            return (a >= b - this) && (a <= b + this);
+        }
+
+        public double GetAbsoluteValue(double ionMz)
+        {
+            switch (Unit)
+            {
+                case Units.ppm:
+                    return ionMz * Value * 1e-6;
+                default:
+                    return Value;
+            }
         }
 
         /// <summary>returns true iff b - a is greater than the value in tolerance (useful for matching sorted mass lists)</summary>
-        public static bool LessThanTolerance(double a, double b, MzTolerance tolerance)
+        public bool LessThanTolerance(double a, double b)
```

Comment:
This method does not seem to be used anywhere. I think this method should be removed, since its name says "Less Than" but the implementation uses "<=".